### PR TITLE
update images and sample types with settingSchema

### PR DESCRIPTION
**Pull request by Mantis-Ranson**
see https://bitbucket.org/technicalrockstars/enebular-product-backlog/issues/254/infotype

Edited both English and Japanese 
- Updated correct sample InfoTypes 
- Updated Images of InfoType

### CODE REVIEW

**Comment by chantlong** [comment]
@k-harada413 gyazoをやめるんじゃなかったっけ？

**Comment by Mantis-Ranson** [comment]
Sorry, just had to make sure plugin.json spelling for `defaultSettings` is correct for sample InfoTypes.

File: ja/GetStarted/index.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ milkcocoa node の **Data Store** は `tutorial`、**Operation** は `Push` で
 DataSource の登録が終わったら、InfoType をアップロードします。今回はサンプルの棒グラフを使います。
 
 <ul>
-  <li><a href="/_asset/sample/sample-bar-chart.zip" target="_blank">サンプル InfoType のダウンロード(zip形式)</a></li>
+  <li><a href="/_asset/sample/barchart-getting-started" target="_blank">サンプル InfoType のダウンロード(zip形式)</a></li>
```
Comment (chantlong) [inline ja/GetStarted/index.md]: Link is broken. Need zip extension.

**Comment by chantlong** [comment]
The link in JA / Getting Started `InfoMotion Type作成のチュートリアルをご覧下さい。` is also broken.

**Comment by Mantis-Ranson** [comment]
Japanese will have to be translated before creating a PR. 
My bad.

**Pull request closed without merging**